### DISCUSSION TITLE
Restriction on Elasticsearch Index

### DISCUSSION THREAD

**Discussion by ishukeshri2712** [jertel/elastalert2#1526] [Q&A] [answered]
Hi @jertel I want to restrict my user to query over only last 10 minutes data  without configuring any alert rule ?

**Comment by jertel** [comment, marked as answer]
ElastAlert 2 has no permission or authorization controls. Any type of restrictions you want to put in place must be done via Kibana, and for help with that I recommend using the Elastic forums.
